docs/developing/ui-component-testing.md

```diff
@@ -0,0 +1,32 @@
+# UI Component Testing
+
+This document gives you a process to follow to check that any UI components
+you have created will be in tip-top shape for the site.
+
+## Browsers to check
+
+It doesn't have to look exactly the same in every browser, but it should look similar and work properly in current versions of these evergreen browsers:
+
+### Desktop
+
+* Firefox
+* Opera
+* Chrome
+* Edge
+* IE 11
+* Safari (latest and latest but one)
+
+### Mobile
+
+Latest versions of:
+
+* iPhone: Chrome and Safari
+* iPad: Chrome and Safari
+* Samsung phone: Chrome and Firefox
+* Google phone: Chrome and Firefox
+
+## Procedure to follow
+
+* Visually check the html of the component.
```

Comment:
I don't understand what this means. Is it about inspecting the DOM? Or just checking the the component renders correctly?

Reply:
@davidcmoulton ... my understanding was checking the HTML was well formed in the DOM - what was your intention for this?